app/workers/vendor_integration_stats_worker.rb

```diff
@@ -0,0 +1,110 @@
+require 'http'
+
+class VendorIntegrationStatsWorker
+  include Sidekiq::Worker
+
+  SLACK_CHANNELS = {
+    'tribal' => 'tribal_dfe_collaboration_',
+    'ellucian' => 'ellucian_dfe_collaboration',
+  }.freeze
+
+  def perform(vendor_name)
+    @vendor_name = vendor_name
+    @webhook_url = fetch_webhook_url
+
+    if @webhook_url.present?
+      message = SlackReport.new(vendor_name).generate
+      post_to_slack message
+    end
+  end
+
+private
+
+  def fetch_webhook_url
+    ENV.fetch "#{@vendor_name.upcase}_INTEGRATION_STATS_SLACK_URL"
```

Comment:
Why do we have to use separate URLs for each vendor?

Reply:
Each url is tied to a different Slack channel

Reply:
ach — they used to be more permissive!